docs/other/fontsobject.rst

```diff
@@ -29,24 +29,39 @@ FontsObject.allFonts
 
 **Description**
 
-The list of all the fonts currently available on your system, sorted by Font Family.
+The list of all the fonts currently available on your system.
+
+They are grouped into what is named a family group which are Arrays of :ref:`fontobject`.
+
+.. Naming and ordering::
+
+The Family Name of the group is simply the ``Font.familyName`` of any of the :ref:`fontobject` in the group.
```

Comment:
- Prefer references for any text that is refers to a documented attribute.
- Prefer the use of labels for references so the sentences can read correctly while still linking to the object entry.

```suggestion
The Family Name of the group is simply the :ref:`familyName <FontObject.familyName>` of any of the :ref:`Font objects<fontobject>` in the group.
```